grammars/go.cson

```diff
@@ -85,7 +85,7 @@
   }
   {
     'name': 'storage.type.go'
-    'match': '\\b(chan|map|bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
+    'match': '\\b(bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
```

Comment:
Even through your absolutely correct that both `chan` and `map` are listed as reserved (key)words, they are also both types (https://golang.org/ref/spec#Map_types and https://golang.org/ref/spec#Channel_types) listed and explained right along side all other types (https://golang.org/ref/spec#Types).

Taking that into account and looking at how the words are used within the language (to define types) I think it makes much more sense to assign the `storage,type.go` scope instead of the `keyword.go` scope.